src/ln/msgs.rs

```diff
@@ -564,8 +603,17 @@ impl MsgEncodable for AcceptChannel {
 }
 
 impl MsgDecodable for FundingCreated {
-	fn decode(_v: &[u8]) -> Result<Self, DecodeError> {
-		unimplemented!();
+	fn decode(v: &[u8]) -> Result<Self, DecodeError> {
+		if v.len() != 32+32+2+64 {
```

Comment:
isn't it a general rule that oversize should be ignored for forward compatibility ?

Reply:
Fixed in a new commit.